modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -32,6 +32,33 @@ export async function getBitgoGpgPubKey(bitgo: BitGoBase): Promise<Key> {
   return await readKey({ armoredKey: bitgoPublicKeyStr });
 }
 
+/**
+ * Verify an Eddsa KeyShare Proof.
+ *
+ * @param senderPubKey public key of the sender of the privateShareProof
+ * @param privateShareProof u value proof
+ * @param uValue u value from an Eddsa keyshare
+ * @return {boolean} whether uValue proof actually was signed by sender as part of their subkeys
+ */
+export async function verifyShareProof(
+  senderPubKey: string,
+  privateShareProof: string,
+  uValue: string
+): Promise<boolean> {
+  const decodedProof = await pgp.readKey({ armoredKey: privateShareProof });
+  const otherOvcGpgKey = await pgp.readKey({ armoredKey: senderPubKey });
+  const isValid = (await decodedProof.verifyPrimaryUser([otherOvcGpgKey]))[0].valid;
```

Comment:
why bother making a variable if you're just going to use it once one line below?   this can just be put into an if statement